### DISCUSSION TITLE
WiFiMulti?

### DISCUSSION THREAD

**Discussion by JAndrassy** [khoih-prog/WiFiNINA_Generic#17] [Ideas]
Khoi, a suggestion for a new library: WiFiMulti for non esp WiFi libraries

**Comment by khoih-prog** [comment]
Thanks for the suggestion. I'll definitely spend some time to study and implement this good new feature.

Just afraid we go to far away from the original `WiFiNINA`. What do you think?

**Reply by JAndrassy** [reply]
It can be a new separate library. It only works with the `WiFi` object

**Reply by khoih-prog** [reply]
OK. New library such as `MultiWiFi_Generic` for all WiFi besides ESP32/ESP8266

**Comment by khoih-prog** [comment]
Hi @jandrassy 

The new [**WiFiMulti_Generic releases v1.0.0**](https://github.com/khoih-prog/WiFiMulti_Generic/releases/tag/v1.0.0) has just been published. Your contribution is noted in [**Contributions and Thanks**](https://github.com/khoih-prog/WiFiMulti_Generic#contributions-and-thanks)

Example can work with your [**WiFiEspAT** Library](https://github.com/jandrassy/WiFiEspAT)

Please have a look, test and have more suggestions,

Thanks and Best Regards,

---

#### Releases v1.0.0

1. Initial coding to support **ESP32, ESP8266, WiFiNINA and ESP8266_AT and ESP32_AT WiFi** for many boards (nRF52, SAMD, Teensy, RP2040, SAM-DUE, ESP32, ESP8266, etc.)
